install.sh

```diff
@@ -260,6 +273,38 @@ set_synchrony_url() {
   fi
 }
 
+# Update the current load balancer listener on port 7999 to use the TCP protocol
+enable_ssh_tcp_protocol_on_lb_listener() {
+  local install_bitbucket
+  local region
+  local load_balancer_dns
+  local load_balancer_name
+  local original_instance_port
+
+  install_bitbucket=$(get_product "bitbucket" "${CONFIG_ABS_PATH}")
+
+  if [ -n "${install_bitbucket}" ]; then
+    region=$(get_variable 'region' "${CONFIG_ABS_PATH}")
+    load_balancer_dns=$(terraform output | grep '"load_balancer_hostname" =' | sed -nE 's/^.*"(.*)".*$/\1/p')
+    load_balancer_name=$(echo "${load_balancer_dns}" | cut -d '-' -f 1)
+    original_instance_port=$(aws elb describe-load-balancers --load-balancer-name af4905e438c1d4ba2ada43c9eda1b879 --query 'LoadBalancerDescriptions[*].ListenerDescriptions[?Listener.LoadBalancerPort==`7999`].Listener[].InstancePort | [0]' --region "${region}")
+
+    log "Enabling SSH connectivity for Bitbucket. Updating load balancer [${load_balancer_dns}] listener protocol from HTTP to TCP on port 7999..."
```

Comment:
I think we can extract `7999` into some constant. There is a little chance it will change soon, but we are repeating this magic number several times.